package.json

```diff
@@ -1,15 +1,14 @@
 {
   "name": "headbreaker",
   "version": "3.0.0",
-  "description": "Jigsaw puzzles library for js",
-  "main": "src/index.js",
-  "types": "dist/headbreaker.d.ts",
+  "type": "module",
   "scripts": {
-    "test": "node_modules/ts-mocha/bin/ts-mocha -p tsconfig.json test/*",
-    "all": "npm test && npm run build && npm run api",
-    "build": "webpack --config webpack.prod.js",
-    "prepublishOnly": "npm run build",
-    "api": "node_modules/jsdoc-import-support/jsdoc.js src/ -d docs/api -R README.md; exit 0"
+    "dev": "vite --config vite.config.website.ts",
+    "build:website": "vite build --config vite.config.website.ts",
+    "build:lib": "vite build --config vite.config.lib.ts",
+    "build": "npm run build:website && npm run build:lib",
+    "preview": "vite preview --config vite.config.website.ts",
+    "test": "vitest"
```

Comment:
Also, we should ensure that we deploy a working `dist`, so that people can not only use it as an npm package, but as a plan old self-contained js file

Reply:
What do you mean? Do you men ONLY deploying the library to dist?

Reply:
> Also, we should ensure that we deploy a working `dist`, so that people can not only use it as an npm package, but as a plan old self-contained js file

Ahh yeah I think that problem is already solved. I think? Just run npm build the library